### PR TITLE
Video figure updates for 2023-02-04 CTF (miscommunication)

### PR DESCRIPTION
**Pull request by GordonBeeming**
[🐛 Fixing video captions on Do you know how to report bugs and give su…](https://www.ssw.com.au/rules/report-bugs-and-suggestions/) 
[🐛 Fixing video figure on Do you know that it's bad to win the argumen…](https://ssw.com.au/rules/lose-battle-keep-client/)
[🐛 Update video figure on Do you find the positive in the decisions?](https://ssw.com.au/rules/the-happiness-equation/)
[🐛 Updating video figures on Do you speak up about unfairness?](https://www.ssw.com.au/rules/speak-up/)

### CODE REVIEW

File: rules/the-happiness-equation/rule.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ guid: f355b232-00cf-4201-8eba-71bf7d12c3c5
 ---
 
 `youtube: https://www.youtube.com/embed/sT3vs48Rt8I`
+**Video: Fairness and Helping Each Other - 10 tips with Adam Cogan (8 min)**
```
Comment (billyterblanche) [inline rules/the-happiness-equation/rule.md:15]:
```suggestion
**Video: Fairness and Helping Each Other - 10 tips with [Adam Cogan](https://ssw.com.au/people/adam-cogan) (8 min)**
```

File: rules/report-bugs-and-suggestions/rule.md [hunk 2]
```diff
@@ -26,14 +26,20 @@ When reporting bugs and giving product feedback, it is essential that you are as
...
-* [Tip #6: Make sure when using backlog, the Product Owner will still get an email](#tip-6-make-sure-when-using-backlog-the-product-owner-will-still-get-an-email)  
-* [Tip #7: Separate PBIs](#tip-7-separate-pbis)  
-* [Tip #8: Use emojis and prefixes for PBI/Issues titles, or email subjects](#tip-8-use-emojis-and-prefixes-for-pbiissues-titles-or-email-subjects)  
+- [Tip #1: Draft your bug with enough details](#tip-1-draft-your-bug-with-enough-details)
+- [Hi team,](#hi-team)
+- [Hi team,](#hi-team-1)
+- [Hi Danny,](#hi-danny)
+- [Hi Danny,](#hi-danny-1)
+- [Expected result](#expected-result)
+- [Actual result](#actual-result)
```
Comment (billyterblanche) [inline rules/report-bugs-and-suggestions/rule.md:35]: What happened here? Is this intentional?

Comment (GordonBeeming) [reply rules/report-bugs-and-suggestions/rule.md:35]: Thanks for catching... it's not... things went weird
 
JudgingYouGIF

**Review by billyterblanche — APPROVED** [review]
🚢

File: rules/report-bugs-and-suggestions/rule.md (hunk 2 above)
Comment (GordonBeeming) [reply rules/report-bugs-and-suggestions/rule.md]: The markdown all in one extension has a auto table of contents which made the formatting weird

image

**Pull request merged**